src/platform_impl/ios/window.rs

```diff
@@ -64,62 +94,47 @@ impl Inner {
     }
 
     pub fn inner_position(&self) -> Result<LogicalPosition, NotSupportedError> {
-        unsafe {
-            let safe_area = self.safe_area_screen_space();
-            Ok(LogicalPosition {
-                x: safe_area.origin.x as _,
-                y: safe_area.origin.y as _,
-            })
-        }
+        self.outer_position()
     }
 
     pub fn outer_position(&self) -> Result<LogicalPosition, NotSupportedError> {
-        unsafe {
-            let screen_frame = self.screen_frame();
-            Ok(LogicalPosition {
-                x: screen_frame.origin.x as _,
-                y: screen_frame.origin.y as _,
-            })
-        }
+        let frame = unsafe {
+            if self.window.is_null() {
+                view::frame_from_window_attributes(&self.window_attributes.borrow())
+            } else {
+                msg_send![self.window, frame]
+            }
+        };
+        Ok(LogicalPosition {
+            x: frame.origin.x,
+            y: frame.origin.y,
+        })
     }
 
-    pub fn set_outer_position(&self, position: LogicalPosition) {
-        unsafe {
-            let screen_frame = self.screen_frame();
-            let new_screen_frame = CGRect {
-                origin: CGPoint {
-                    x: position.x as _,
-                    y: position.y as _,
-                },
-                size: screen_frame.size,
-            };
-            let bounds = self.from_screen_space(new_screen_frame);
-            let () = msg_send![self.window, setBounds: bounds];
-        }
+    pub fn set_outer_position(&self, _position: LogicalPosition) {
+        warn!("`Window::set_outer_position` is ignored on iOS")
     }
 
     pub fn inner_size(&self) -> LogicalSize {
-        unsafe {
-            let safe_area = self.safe_area_screen_space();
-            LogicalSize {
-                width: safe_area.size.width as _,
-                height: safe_area.size.height as _,
-            }
-        }
+        self.outer_size()
```

Comment:
I've tried to address the reasoning behind this change in #1122. The return value of this function doesn't necessarily affect rendering. When resizing the swapchain, gfx-hal applications generally call `compatibility` and use the `current_extent` (which returns the bounds of the `CAMetalLayer`) instead of calling out to winit and using the `inner_size` of the window.